eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -719,6 +719,12 @@
           "code" : "java.method.removed",
           "old" : "method com.azure.resourcemanager.cosmos.models.DatabaseAccountCreateUpdateParameters com.azure.resourcemanager.cosmos.models.DatabaseAccountCreateUpdateParameters::withDatabaseAccountOfferType(java.lang.String)",
           "justification": "The property is a constant, which should not be changed."
+        },
+        {
+          "ignore": true,
+          "code" : "java.method.removed",
+          "old" : "method com.azure.resourcemanager.storage.models.StorageAccountCheckNameAvailabilityParameters com.azure.resourcemanager.storage.models.StorageAccountCheckNameAvailabilityParameters::withType(java.lang.String)",
+          "justification": "The property is now final."
```

Comment:
see above "The property is a constant, which should not be changed."

Reply:
Fixed by [05e86c6](https://github.com/Azure/azure-sdk-for-java/pull/44179/commits/05e86c6c35fe695bce278a013b06c5ff21f918bc)